web/concrete/src/Console/Command/ServiceCommand.php

```diff
@@ -0,0 +1,148 @@
+<?php
+namespace Concrete\Core\Console\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Concrete\Core\Service\ServiceInterface;
+use Core;
+use Exception;
+
+final class ServiceCommand extends Command
+{
+    protected function configure()
+    {
+        $this
+            ->setName('c5:service')
+            ->setDescription('Check or update the web server configuration')
+            ->addOption('service-version', 'r', InputOption::VALUE_REQUIRED, 'The specific version of the web server software', '')
+            ->addArgument('service', InputArgument::REQUIRED, 'The web server to use (apache|nginx)')
+            ->addArgument('operation', InputArgument::REQUIRED, 'The operation to perform (check|update)')
+            ->setHelp(<<<EOT
+Return codes for the check operation:
+  0 operation completed successfully
+  1 errors occurred
+  2 web server configuration is not aligned
+
+Return codes for the update operation:
+  0 operation completed successfully
+  1 errors occurred
+EOT
+            )
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $rc = 0;
```

Comment:
what is `rc`? I would not use an abbreviation here.
Abbreviations make it really hard for new contributors to understand parts of a codebase.